src/aind_data_transfer/transformations/metadata_creation.py

```diff
@@ -130,3 +135,36 @@ def ephys_job_to_subject(
         else:
             logging.error("No data retrieved!")
             return None
+
+
+class DataDescriptionMetadata:
+    """Class to handle the creation of the processing metadata file."""
+
+    output_file_name = "data_description.json"
```

Comment:
With the latest update to aind-data-schema, you can probably just call this as something like
```
output_file_name = RawDataDescription._get_default_name()
```
or something like that.